roles/k8s_addons/tasks/main.yml

```diff
@@ -2,20 +2,39 @@
 # tasks file for k8s-addons
 
 # Helm install
-- import_tasks: helm.yml
 - import_tasks: efs.yml
 - import_tasks: autoscaler.yml
 - import_tasks: datadog.yml
 
 
+- name: Delete secret (S3 IAM account credentials)
```

Comment:
Delete because the secret before was never overwritten with updated credentials.